Sources/OpenPass/DeviceAuthorizationFlow.swift

```diff
@@ -118,6 +130,29 @@ public final class DeviceAuthorizationFlow {
             } catch OpenPassError.tokenAuthorizationPending {
                 // Keep polling
                 continue
+            } catch {
+                Task<Void, Never> {
+                    if case OpenPassError.tokenExpired = error {
+                        try? await openPassClient.recordEvent(
+                            .init(
+                                clientId: openPassClient.clientId,
+                                name: "device_flow_token_expired",
+                                message: "Token expired",
+                                eventType: .info
+                            )
+                        )
+                    } else {
+                        try? await openPassClient.recordEvent(
```

Comment:
Are we missing something that would cover `device_flow_token_verification_failure`?